app/assets/javascripts/components/import_ratings/_modal.html

```diff
@@ -120,6 +120,54 @@ <h2>CSV</h2>
         </div>
       </div>
     </uib-tab>
+    <uib-tab index="2" heading="Snapshots" select="ctrl.clearSelection()">
+      <p>
+        Importing snapshots is fun!  Any missing queries will be created.
```

Comment:
Can we add a sentence about "Ignore the case id in the csv file, it is replaced with this case id"